app/assets/javascripts/controllers/container_dashboard/container_dashboard_controller.js

```diff
@@ -142,6 +142,22 @@ miqHttpInject(angular.module('containerDashboard', ['ui.bootstrap', 'patternfly'
                                                "dates",
                                                $scope.networkUtilizationDailyConfig.units);
           $scope.networkUtilizationLoadingDone = true;
+
+          // Pods metrics
+          $scope.podsUtilizationHourlyConfig = chartsMixin.chartConfig.hourlyPodsUsageConfig;
+
+          if (data.hourly_pods_metrics != undefined) {
+              data.hourly_pods_metrics.xData = data.hourly_pods_metrics.xData.map(function (date) {
+                  return dashboardUtilsFactory.parseDate(date)
+              });
+          }
+
+          $scope.hourlyPodsUtilization =
+              chartsMixin.processUtilizationData(data.hourly_pods_metrics,
+                  "dates",
+                  $scope.podsUtilizationHourlyConfig.units);
+
+          $scope.podsUtilizationLoadingDone = true;
```

Comment:
It suppose to be _creation and deletion_ , Once https://github.com/ManageIQ/manageiq/pull/6856 will be merged.